Userland/Libraries/LibWeb/HTML/HTMLScriptElement.h

```diff
@@ -94,7 +94,7 @@ class HTMLScriptElement final : public HTMLElement {
     JS::GCPtr<DOM::Document> m_preparation_time_document;
 
     // https://html.spec.whatwg.org/multipage/scripting.html#script-force-async
-    bool m_force_async { false };
+    bool m_force_async { true };
```

Comment:
If you're setting force async to `true` here I think it might be worthwhile setting force async to `false` when an async attribute is added to the script element, which, as far as I can see, we aren't currently doing this.

Relevant spec link: https://html.spec.whatwg.org/multipage/scripting.html#script-processing-model:script-force-async

Currently, this script gives the wrong value:
```html
<!DOCTYPE html>
<script>
    const scriptElement = document.createElement("script");
    scriptElement.setAttribute("async", "true");
    scriptElement.removeAttribute("async");
    console.log(scriptElement.async);
</script>
```

Reply:
Good spot - I missed that when reading the spec :^)